dir.props

```diff
@@ -21,8 +21,10 @@
   <PropertyGroup>
     <!-- Temporarily omit code analysis -->
     <SkipImportCodeAnalysisTargets>true</SkipImportCodeAnalysisTargets>
+
     <!-- Hacky BuildConfiguration -->
-    <BuildConfiguration Condition="'$(BuildConfiguration)' == ''">netcoreapp1.1-$(OSEnvironment)</BuildConfiguration>
+    <BuildConfiguration>netcoreapp1.1-$(OS)</BuildConfiguration>
```

Comment:
I think this property should be defined lower right above where we import the generated config props. Also here is what I'm thinking we should default it to. 

```
  <PropertyGroup>
    <ArchGroup Condition="'$(ArchGroup)'==''">x64</ArchGroup>
    <BuildConfiguration Condition="'$(BuildConfiguration)' == ''">netcoreapp-$(OSEnvironment)-Debug-$(ArchGroup)</BuildConfiguration>
  </PropertyGroup>
```
Although we could end up using $(OS) instead of $(OSEnviroment) if needed for some reason.